Xamarin.Forms.Core/SwipeItem.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.ComponentModel;
+using System.Windows.Input;
+
+namespace Xamarin.Forms
+{
+	public interface ISwipeItem
+	{
+		ICommand Command { get; set; }
+		object CommandParameter { get; set; }
+
+		event EventHandler<SwipeItemInvokedEventArgs> Invoked;
+	}
+
+	public class SwipeItem : ContextItem, ISwipeItem
+	{
+		public static readonly BindableProperty BackgroundColorProperty = BindableProperty.Create(nameof(BackgroundColor), typeof(Color), typeof(SwipeItem), Color.Default);
+
+		public Color BackgroundColor
+		{
+			get { return (Color)GetValue(BackgroundColorProperty); }
+			set { SetValue(BackgroundColorProperty, value); }
+		}
+
+		public event EventHandler<SwipeItemInvokedEventArgs> Invoked;
+
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public void OnInvoked() => Invoked?.Invoke(this, new SwipeItemInvokedEventArgs(this));
```

Comment:
as the sender is the swipe item, there's no need to pass it again as eventargs

Reply:
what does trigger the command ?

Reply:
Right. I changed the `EventArgs`. The Command is triggered from the Renderers.

Reply:
the command and the event should be triggered at the same time. OnInvoked should trigger it

Reply:
I have applied changes trigger both at the same time.